components/serializer.rst

```diff
@@ -413,6 +413,80 @@ As for groups, attributes can be selected during both the serialization and dese
 Ignoring Attributes
 -------------------
 
+Set `false` by default, you can ignore attribute by two methods.
```

Comment:
I don't understand this -> "Set `false` by default, ..."

Maybe we can reword it like this:

```rst
All attributes are included by default when serializing objects. You have
two alternatives to ignore some of those attributes.
```